ci-conda.Dockerfile

```diff
@@ -164,10 +165,6 @@ RUN cat /tmp/condarc.tmpl | envsubst | tee /opt/conda/.condarc; \
 
 RUN /opt/conda/bin/git config --system --add safe.directory '*'
 
-# Install CI tools using pip
-RUN pip install dunamai "rapids-dependency-file-generator==1.*" \
```

Comment:
was `dunamai` intentionally removed here? I think we still need that

Reply:
though it looks like that can be fetched from conda too:

- https://anaconda.org/conda-forge/dunamai

Reply:
Updated to get dunamai from Conda.